modules/installation-azure-service-principal.adoc

```diff
@@ -177,87 +173,37 @@ output. You need these values during {product-title} installation.
 +
 [source,terminal]
 ----
-$ az ad sp create-for-rbac --role Contributor --name <service_principal> <1>
+$ az ad sp create-for-rbac --role Contributor --name <service_principal> \ <1>
+  --scopes /subscriptions/<subscription_id> <2>
 ----
-<1> Replace `<service_principal>` with the name to assign to the service principal.
+<1> Specify the service principal name.
+<2> Specify the subscription ID.
 +
 .Example output
 [source,terminal]
 ----
-Changing "<service_principal>" to a valid URI of "http://<service_principal>", which is the required format used for service principal names
-Retrying role assignment creation: 1/36
-Retrying role assignment creation: 2/36
-Retrying role assignment creation: 3/36
-Retrying role assignment creation: 4/36
+Creating 'Contributor' role assignment under scope '/subscriptions/<subscription_id>'
+The output includes credentials that you must protect. Be sure that you do not include these credentials in your code or check the credentials into your source control. For more information, see https://aka.ms/azadsp-cli
 {
-  "appId": "8bd0d04d-0ac2-43a8-928d-705c598c6956",
-  "displayName": "<service_principal>",
-  "name": "http://<service_principal>",
-  "password": "ac461d78-bf4b-4387-ad16-7e32e328aec6",
-  "tenant": "6048c7e9-b2ad-488d-a54e-dc3f6be6a7ee"
+  "appId": "ac461d78-bf4b-4387-ad16-7e32e328aec6",
+  "displayName": <service_principal>",
+  "password": "00000000-0000-0000-0000-000000000000",
+  "tenant": "6047c7e9-b2ad-488d-a54e-dc3f6be6a7ee"
```

Comment:
how about change to 8049c7e9-c3de-762d-a54e-dc3f6be6a7ee,  to match with above show